### ISSUE TITLE
[ACTIVE_STALE_TAKEOVER] D03|ARXIV|LOCAL_FREE-EXECUTION|V2 — run real bounded payload

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1357]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1357
INSTANCE_TAG: SOL-SCI-CLOSER-20260912T0001+02-REAL-EXEC
SWARM_LANE_KEY: D03|ARXIV|LOCAL_FREE-EXECUTION|V1
PREFERRED_LANE: #9 (D08)
PREFERRED_WORK_KIND: REDTEAM_AUDIT
ROTATION_REASON: preferred D08/red-team work is downstream of the current zero-exposure data gate and has live owners; rotate to the nearest finish-first real-execution package on the learned-20M critical path.
PARENT_ISSUES: #4 #548 #723 #769 #871 #879 #998 #1020
STATUS: ACTIVE_STALE_TAKEOVER_PENDING_POSTCLAIM_DIRECT_ARBITRATION
BASE_SHA: 44581b0779f94f2a2a614a73c3446d70666b64ae
CLAIMED_AT_UTC: 2026-09-11T22:05:00Z
LEASE_UNTIL_UTC: 2026-09-12T04:05:00Z
CI_PRESSURE_AT_CLAIM: GREEN (0 queued + 0 in_progress)

STALE_TAKEOVER EVIDENCE:
- prior same-semantic owner #1020 explicitly leased only through 2026-09-10T18:55:00Z;
- fresh OPEN semantic search before claim returned #1020 and no newer ArXiv execution successor;
- live main still contains the exact known execution blocker in the integrated ArXiv materializer: `EMAIL_RE = ... re.I`;
- historical #879 scientific execution ran focused tests then failed at Ruff FURB167 before source download/materialization, so REAL_DATA_EXECUTED remains false for this line;
- current Actions pressure is zero queued / zero in-progress; no active CI is plausibly owned by #1020;
- this takeover continues the integrated incumbent implementation only; it creates no second ArXiv parser/data family.

CANDIDATE COMPARISON:
1. PR #903 CPython real execution — yielded because newer same-semantic owner #1356 is live.
2. Integrated ArXiv #879 real execution — SELECTED: mechanics already canonical, stale owner, one concrete behavior-preserving lint unblock, and a pinned 232,616,926-byte immutable shard ready for real two-build LOCAL_FREE execution.
3. PR #886 Public Domain Review — stale fallback, but broader two-file execution surface.
4. USGPO #1001 — stale fallback, but materially more implementation remains before real execution.

OWNED_SURFACES:
- current-main `tools/materialize_d03_common_pile_arxiv_abstracts_v1.py`, its focused tests/config only as required by observed execution defects;
- behavior-preserving `re.I -> re.IGNORECASE` repair first;
- exact incumbent source contract: `common-pile/arxiv_abstracts@46de78c48636c0b46f60049dfd1c5a3710d233f9`, file `00003_arxiv-abstracts.jsonl.gz`, bytes 232616926, SHA-256 `3d781bf7617fd9a6840211227c6c8831280dba457a6d9c79a2a5a357e025a8b7`;
- real LOCAL_FREE two-acquisition/two-materialization execution using existing mechanics; require source hash/size verification and byte-identical candidate/report identities or retain exact negative evidence;
- compact text-free evidence only; raw/candidate payload is not committed;
- one current-main repair PR maximum, shared CI only; no permanent/new worker workflow.

AVOID_SURFACES:
- no central global dedup/decontamination/privacy/balance/split/packing/tokenizer/readiness/model/trainer/evaluation/final-test mutation;
- no alternate ArXiv materializer or source-family architecture;
- no training or optimizer update while optimized-target exposure authority is zero;
- no paid compute, foreign pretrained weights, or external LLM/API corpus intelligence;
- no merge while live integration owner #1320 lease is active.

OBJECTIVE:
Remove the exact execution blocker on the already-integrated ArXiv intake, then perform genuine bounded LOCAL_FREE source execution twice and publish only reproducibility/evidence needed for downstream corpus composition. A negative real execution result is terminal evidence and must not be rewritten as success.

PACKAGE_DIMENSIONS:
- implementation_or_primary_research
- focused_tests
- adversarial_or_negative_tests
- machine_readable_evidence_or_validator
- live_authority_binding
- end_to_end_or_integration_proof
- measured_benchmark_or_reproducibility_proof
- documentation_or_operator_handoff

ACCEPTANCE:
1. mandatory postclaim direct paginated OPEN issue collection finds no earlier still-live successor beyond stale #1020; otherwise abandon/yield before Product mutation;
2. direct open PR collection shows no newer ArXiv Product owner/branch conflict;
3. exact main blocker is repaired without changing filter semantics;
4. focused tests/tool Ruff pass on the exact repair head;
5. real pinned 232,616,926-byte source is acquired twice under LOCAL_FREE, exact size/SHA verified before parsing, and two deterministic materializations compare byte-identically, OR exact external/runtime failure is durably reported;
6. retained evidence remains text-free and states rights `REVIEW_REQUIRED`, training/evaluation/canonical/family credit zero, unique-loss/optimized-target authority zero;
7. final exact-head shared CI SUCCESS before independent audit handoff;
8. no tokenizer fit, optimizer/model update, learned-weight claim, final-test access, or paid compute.

FALLBACK:
If arbitration loses, mark ABANDONED_DUPLICATE and rotate this same issue. If source network/runtime blocks the real execution after a valid repair, preserve the exact blocker and rotate to another independent capacity gate rather than fabricate yield.

TRUTH_BOUNDARY_AT_CLAIM:
REAL_DATA_EXECUTED_ON_THIS_LINE=false
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
OPTIMIZER_UPDATES_EXECUTED=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
TOKENIZER_FIT_AUTHORIZED=false
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

**Comment by Oleksii-debug** [comment]
CHECKPOINT 2026-09-11T22:21Z — PRODUCT TERMINAL / AUDIT REQUIRED

- Stale #1119 takeover arbitration completed before Product mutation: direct OPEN issues paginated newest-first through exact empty page 10; direct OPEN PRs through exact empty page 6; no newer same-semantic ArXiv source-rights owner/Product.
- Created one Product branch `d03/arxiv-rights-1357` from exact main `44581b0779f94f2a2a614a73c3446d70666b64ae`; one PR only: #1361.
- Final exact head: `cbec5d66e493ee5fa04a67d137c05a8217677d74`.
- Shared CI `34653522546` COMPLETED/SUCCESS; job `103440842092` SUCCESS; workflow budget/Ruff/full pytest all PASS.
- PR #1361 is open, non-draft, mergeable=true.
- Frozen policy identity: `b667a71cf476282d45476babc71b9d401ffdaaa38e03b9fac8a6319614b57442`.
- Source decision is narrow `CONDITIONAL_SOURCE_ADMISSION` for exact ArXiv descriptive-metadata/abstract records retaining exact CC0 metadata license; full e-print rights are not inferred.
- Truth remains: payload source admission not executed by this package; corpus/family/training/unique-loss/exposure/tokenizer/evaluation/final-test/model-training credit remains zero/false.

STATUS: READY_FOR_INDEPENDENT_AUDIT. Product mutation stops until a fresh different worker returns an exact-head verdict; no self-certification or merge.
